src/components/coin-icon/CoinIcon.js

```diff
@@ -35,7 +35,7 @@ const CoinIcon = ({
         color={color}
         fallbackRenderer={CoinIconFallback}
         forceFallback={forceFallback}
-        shadowColor={tokenMetadata?.extensions?.shadowColor || color}
```

Comment:
I have now mapped RAINBOW_TOKEN_LIST to unwrap all parameters in extensions so we can access them more naturally (token metadata will be based on this new RAINBOW_TOKEN_LIST)